dev-libs/libxml++/libxml++2-2.42.3.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="C++ wrapper for libxml"
+DESCRIPTION="libxml++ (a.k.a. libxmlplusplus) provides a C++ interface to XML files. \
+It uses libxml2 to access the XML files.
+
+libxml++-2.6, libxml++-3.0, libxml++-4.0 and libxml++-5.0 are different \
+parallel-installable ABIs. This package contains libxml++-2.6."
+HOMEPAGE="https://libxmlplusplus.sourceforge.net/"
+COPYRIGHT="2004-2024 LibXML++"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/libxml++/2.42/libxml++-$portVersion.tar.xz"
+CHECKSUM_SHA256="74b95302e24dbebc56e97048e86ad0a4121fc82a43e58d381fbe1d380e8eba04"
+SOURCE_FILENAME="libxml++-$portVersion.tar.bz2"
+SOURCE_DIR="libxml++-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="2.0.7"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libxml++2$secondaryArchSuffix = $portVersion
+	lib:libxml++_2.6$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libglibmm$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libxml++2${secondaryArchSuffix}_devel = $portVersion
+	devel:libxml++_2.6$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	libxml++2$secondaryArchSuffix == $portVersion base
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libglibmm$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libglibmm$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:meson
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p buildmeson
+
+	# The "meson setup [options]" command uses setup as a folder?
+	# Still throws a warning about it though
+	meson buildmeson --prefix $prefix \
+		--buildtype=release
```

Comment:
```suggestion
		--buildtype=release \
		--libdir=$libDir
```

Reply:
Ok, done, also made the includedir change we talked about. Of course, I have no way of knowing if that worked for x86